containers/buildUbuntu6.docker/Dockerfile

```diff
@@ -14,7 +14,7 @@ RUN apt-get update --fix-missing && \
 
 ENV DEBIAN_FRONTEND=noninteractive
 
-RUN apt-get update --fix-missing && apt-get install -y build-essential gcc-${COMPILER_VERSION} g++-${COMPILER_VERSION} cmake make bison flex python3 ccache git libjemalloc-dev vim exuberant-ctags gdb fish psmisc sudo debhelper debconf jq wget libdb-dev curl gnupg2 gcovr prometheus bc tcpdump && apt-get clean
+RUN apt-get update --fix-missing && apt-get install -y build-essential gcc-${COMPILER_VERSION} g++-${COMPILER_VERSION} cmake make bison flex python3 ccache git libjemalloc-dev vim exuberant-ctags gdb fish psmisc sudo debhelper debconf jq wget libdb-dev curl gnupg2 gcovr prometheus bc tcpdump liburing2 liburing-dev && apt-get clean
```

Comment:
Is liburing2 necessary?

Reply:
In case of Ubuntu if a build is done within this container we use shared linking by default. This is just for backward compatibility.

Reply:
Can we use static? 
rocksdb by default find only static liburing, we made patch to find shared in maintainer mode (mostly for developers local machines), but in general I don't think it's good idea to rely on it in CI

Reply:
https://github.com/arangodb/oskar/pull/542/commits/11eda8c33e359e32ae6f162f43725bbfe6a55813